### PR TITLE
chore(test): define the correct provider type locator

### PR DESCRIPTION
**Pull request by amisskii**
Signed-off-by: Anton Misskii <amisskii@redhat.com>

### What does this PR do?

Changes the Kind provider type locator 

### What issues does this PR fix or reference?

[](https://github.com/containers/podman-desktop/issues/9483)

### How to test this PR?

- [ ] Tests are covering the bug fix or the new feature

### CODE REVIEW

File: tests/playwright/src/model/pages/create-kind-cluster-page.ts [hunk 1]
```diff
@@ -76,14 +81,12 @@ export class CreateKindClusterPage extends BasePage {
...
-      if (providerTypeOptions.includes(providerType)) {
-        await this.providerTypeCombobox.selectOption({ value: providerType });
-        await playExpect(this.providerTypeCombobox).toHaveValue(providerType);
-      } else {
-        throw new Error(`${providerType} doesn't exist`);
-      }
+      await playExpect(this.providerType).toBeVisible();
+      await this.providerType.click();
+      const providerTypeOption = this.page.getByRole('button', { name: providerType });
+      await playExpect(providerTypeOption).toBeVisible();
```
Comment (cbr7) [inline tests/playwright/src/model/pages/create-kind-cluster-page.ts]:
```suggestion
      await playExpect(this.providerType).toBeEnabled();
      await this.providerType.click();
      const providerTypeOption = this.page.getByRole('button', { name: providerType, exact: true });
      await playExpect(providerTypeOption).tobeEnabled();
```

**Comment by cbr7** [comment]
@amisskii Looks like you didn't update the relevant unit tests when you made the change to the dropdown component.

**Comment by amisskii** [comment]
@cbr7, there were two elements with the same aria-label. `Error: TestingLibraryElementError: Found multiple elements with the text of: record-description`. I’ve fixed that.

**Review by cbr7 — APPROVED** [review]

**Comment by amisskii** [comment]
https://github.com/containers/podman-desktop-e2e/actions/runs/11455558328

**Review by benoitf — CHANGES_REQUESTED** [review]
need to be a separate PR with a test

File: packages/ui/src/lib/dropdown/Dropdown.svelte [hunk 2]
```diff
@@ -191,6 +191,7 @@ function onWindowClick(e: Event): void {
     class:group-hover-placeholder:text-[color:var(--pd-input-field-placeholder-text)]={!disabled}
     disabled={disabled}
     id={id}
+    aria-label={`${ariaLabel} Button`}
```
Comment (benoitf) [inline packages/ui/src/lib/dropdown/Dropdown.svelte:194]: need a unit test

**Comment by benoitf** [comment]
the title of this PR is misleading, it's not only something in the tests,

you need to provide first a PR to the UI library and with a unit test to the component and get approved there

then, you can do the changes in tests

**Comment by benoitf** [comment]
revert PR is there https://github.com/containers/podman-desktop/pull/9530

**Pull request merged**